MovieJournalMVC/Model/Store.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Store.swift
+//  MovieJournalMVC
+//
+//  Created by Lauren Small on 3/12/19.
+//  Copyright © 2019 Lauren Small. All rights reserved.
+//
+
+import Foundation
+
+class Store {
+    static let shared: Store = Store()
+    
+    var movieReviews: [MovieReview]
+    
+    private init(){
```

Comment:
You'll need to make this initializer non-private to be able to use it in tests.

Reply:
private init() is gone!